ucm2/Toradex/apalis/eval-HiFi.conf

```diff
@@ -0,0 +1,13 @@
+# Use case configuration for Toradex Apalis iMX8, Evaluation Board v1.2
+
+SectionDevice."Headphones" {
+	Comment "Headphones"
+
+	EnableSequence [
+		cset "name='Headphone Switch' on"
+	]
+
+	DisableSequence [
+		cset "name='Headphone Switch' off"
+	]
+}
```

Comment:
Overall: Missing PCM device specifications (Values) ....

Reply:
I added the `Value` node, please let me know... thanks!